gitlab/gitlab_exporter.py

```diff
@@ -124,6 +125,8 @@ def get_all_merge_requests_from_gitlab():
             }
 
             create_entity('mergeRequest', entity, access_token=token)
+        
+        print(f"Created {merge_requests.__len__()} merge requests")
```

Comment:
It is better to print the number of found microservices and merge requests, and then separately count the number of entities that the script succeeded in creating.

In the end you will be able to print "created X microservice entities out of Y queried from GitLab"